cli/cmd/app/deploy/deploy_test.go

```diff
@@ -32,32 +29,36 @@ func TestDeploy(t *testing.T) {
 		apps.On("DeployApp", mock.Anything, mock.Anything).Return(app.DeployAppOutput{DeploymentVersionID: deployVersionID}, nil)
 		apps.On("DeployEvents", mock.Anything, mock.Anything).Return(nil)
 
-		err := Deploy(context.TODO(), apps, appDir, "", slug, appName, false)
+		return apps
+	}
+
+	t.Run("give no existing app then happy path can run", func(t *testing.T) {
```

Comment:
Seems like this and the next test are identical tests with different names

Reply:
Nice catch! It was a mistake in the refactoring of the setup.

Reply:
96d59abc065010c4506235feece548a246ed869e